Makefile

```diff
@@ -352,6 +353,7 @@ config: .env ## Run first-time configuration
 	install_portainer=$$(readConfigVar "Install portainer? (yes|no)" "install_portainer" "$(install_portainer)")
 	internal_managed_cc_tls=$$(readConfigVar "Install portainer? (yes|no)" "internal_managed_cc_tls" "$(internal_managed_cc_tls)")
 	whitelist_ip_file=$$(readConfigVar "whitelist file for external" "whitelist_ip_file" "$(whitelist_ip_file)")
+	vpn_client_ip_file=$$(readConfigVar "whitelist file for external" "vpn_client_ip_file" "$(vpn_client_ip_file)")

```

Comment:
change the description for this var

Reply:
I will add appropriate description like
vpn_client_ip_file=$$(readConfigVar "vpn client ip file for static routing, the IP prefixes of client private network" "vpn_client_ip_file" "$(vpn_client_ip_file)")